defaults/main.yml

```diff
@@ -7,6 +7,7 @@ mssql_version: null
 mssql_upgrade: false
 mssql_password: null
 mssql_edition: null
+mssql_tools_versions: [17, 18]
```

Comment:
By default you install both versions?

Reply:
oops, didn't update it after doing my test.